documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
+1) Maintaining some states over clock ticks to retain important information that will decide algorithm's next action 
```

Comment:
Markdown doesn't accept `1) 2)` as valid numbered list notation. As a result... in the rendered result it all becomes jumbled into one large paragraph. You should stick to `1. 2.` instead.

Also, it'd help to use a WYSIWYG markdown editor like Macdown so you know you're using the proper syntax.